.github/workflows/push.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           runTestsParameters: >-
             -d zend_extension=opcache.so
             -d opcache.enable_cli=1
-            ${{ !matrix.asan && '-d opcache.jit_buffer_size=16M' || '' }}
+            ${{ matrix.asan && '-d opcache.jit=disable' || '-d opcache.jit=tracing' }}
```

Comment:
This would result in `opcache.jit=tracing` twice for non-asan, as it's already added in the test action.

Reply:
Hmm, where? I see only two instances of opcache.jit=tracing in this workflow, one here and the other in the mac OS tests...